lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/PureContext.scala

```diff
@@ -19,6 +19,11 @@ object PureContext {
   val err           = LazyVal(EitherT(nothingCoeval))
   val errRef        = "throw"
 
+  val fraction: PredefFunction = PredefFunction("fraction", 1, LONG, List(("value", LONG), ("numerator", LONG), ("denominator", LONG))) {
+    case (v: Long) :: (n: Long) :: (d: Long) :: Nil => Right((BigInt(v) * n / d).toLong)
```

Comment:
can toLong overflow? please add tests that 

fraction (Long.MaxValue, 2,  3) doesn't overflow,
fraction (Long.MaxVaue,3,2) overflows (doesn't return negative)